ledger/block/src/transition/output/mod.rs

```diff
@@ -168,7 +180,7 @@ impl<N: Network> Output<N> {
             Output::Public(hash, Some(output)) => {
                 match output.to_fields() {
                     Ok(fields) => {
-                        // Construct the (console) output index as a field element.
+                        // Construct the (public) output index as a field element.
```

Comment:
I believe these "(console)" comments are not referring to the Output type, but that the output itself is in console-land.